src/openscm_runner/adapters/fair_adapter/_scmdf_to_emissions.py

```diff
@@ -214,6 +214,35 @@ def scmdf_to_emissions(
 
     for var_df in scmrun.groupby("variable"):
         variable = var_df.get_unique_meta("variable", no_duplicates=True)
+        # for scenarios providing both aggregate fossil and AFOLU CO2, ignore the total
+        if variable.endswith("|CO2"):
+            continue
+        # we don't care about the split for most emitted species
+        if variable.endswith(
```

Comment:
```suggestion
        if "MAGICC AFOLU|" in variable or "MAGICC Fossil and Industrial|" in variable
```

should do the same thing and means you only need one line